roles/openshift_logging_elasticsearch/vars/default_images.yml

```diff
@@ -1,5 +1,5 @@
 ---
 __openshift_logging_elasticsearch_image_prefix: "{{ openshift_logging_image_prefix | default('docker.io/openshift/origin-') }}"
-__openshift_logging_elasticsearch_image_version: "{{ openshift_logging_image_version | default('latest') }}"
+__openshift_logging_elasticsearch_image_version: "{{ openshift_logging_image_version | default(openshift_image_tag) }}"
```

Comment:
These are for origin, we should not be changing these as they _should_ default to `latest`

Reply:
That makes sense for latest Origin, but would it work with Origin 3.6? @fabianvf is also not sure if `latest` is a good idea generally - see https://github.com/openshift/openshift-ansible/pull/6712#discussion_r161204830

Reply:
We should have a latest image for that... but also if we're supporting N-2 3.6 falls outside of that range...
@sdodson thoughts?